.chloggen/add-obsreport.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: 'prometheusremotewritereceiver'
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: obsreport provides unified and consistent observability signals ( metrics, tracing, etc) for components of the OpenTelemetry collector.
```

Comment:
This note doesn't seem to clarify what benefits the user is getting. Obsreport is a concept that lives inside the codebase, but the user will never understand what this is about 😅 .

Could we describe exactly what the user can expect? Is it new metrics, new logs? What is it?